perception/realsense_driver/python/realsense_wrapper/realsense_d435.py

```diff
@@ -51,12 +51,38 @@ def get_intrinsics(self):
     def get_num_cameras(self):
         return len(self.device_ls)
 
-    def get_images(self):
+    def get_images(self, depth=False):
```

Comment:
1. The process for retrieving rgb and depth images are quite independent from each other. Might be easier to just have two separate APIs (`get_rgb_images` and `get_depth_images`), especially since having `depth=True` changes the number of output arguments in the current implementation.
2. A lot of duplicated code between retrieving rgb images and depth images. Maybe we can have a base method `_get_images(self, type=RGB / DEPTH)` (with enums for RGB and DEPTH)?

Reply:
I think the main thing is that we want the RGB + D images to correspond to each other, which calling multiple methods may not result in. Discussing with @0wu we decided on just returning a whole named tuple of [rgb, d, pointcloud] which is aligned & sync'd.